### DISCUSSION TITLE
AMDC - DNP component

### DISCUSSION THREAD

**Discussion by Almenau14** [Severson-Group/AMDC-Hardware#246] [General]
Hello, I have already sent the Gerber files to PCBWAY and I'm waiting for them to finalize the assembly of the components. However, they questioned me about line 56 of the BOM 'https://github.com/Severson-Group/AMDC-Hardware/blob/develop/REV20231005F/BOM_RevF.csv' (9 311-0.0ERCT-ND R47, R93A, R93B, R93C, R93D, R94A, R94B, R94C, R94D Resistor DNP 1206-RES). They asked me the following question: "We bought the components RC1206JR-070RL, but found that you noted DNP in the BOM.". What should I respond to PCBWAY?

The resistor R47 is located on page 3, while the resistors R93 and R94 are on page 17 of the file 'Schematics_RevF.pdf.' Therefore, I could not find the resistors R93A, R93B, R93C, R93D, R94A, R94B, R94C, and R94D in 'Schematics_RevF.pdf.

![DNP-01](https://github.com/user-attachments/assets/353a765f-123f-44a6-997e-e9c32e629ff2)
![DNP-02](https://github.com/user-attachments/assets/4db6a5fd-68bc-4547-8a3d-51e84a4aa65a)

**Comment by npetersen2** [comment]
The `DNP` resistors mean "Do Not Populate" --- you can tell the manufacturer that these should NOT be placed on the board.

They can send them to you separate, and you can rework the board to populate them if you need.
